src/main/java/com/abramchik/taskOnePatterns/behavioral/chainOfResponsibility/Notifier.java

```diff
@@ -0,0 +1,24 @@
+package com.abramchik.taskOnePatterns.behavioral.chainOfResponsibility;
+
+public abstract class Notifier {
+    private int priority;
+    private Notifier nextNotifier;
+
```

Comment:
Also, we can use @fieldsdefaults for decreasing repeatable code

Reply:
done